compiler/passes/docs.cpp

```diff
@@ -452,6 +446,23 @@ void printModule(std::ofstream *file, ModuleSymbol *mod, std::string name) {
   }
 }
 
+void printGlobal(std::ofstream *file, VarSymbol *global, bool config) {
+  if (!global->hasFlag(FLAG_NO_DOC)) {
+    printTabs(file);
+    *file << outputMap[config ? "config" : "global"];
```

Comment:
I could add an individual entry for consts, params, vars, and types but since I don't expect it to vary between text-mode and rst-mode, I don't think it's necessary.